src/DataBase/EntryDatabase.cs

```diff
@@ -178,7 +181,45 @@ public async Task<IEntry> ReadAsync(Expression<Func<IEntryModel, bool>> filter,
 
         public async Task<IEntryList> ReadListAsync(int userId, EntryType entryType)
         {
-            return await ReadListAsync(x => true, userId, entryType);
+            //return await ReadListAsync(x => true, userId, entryType);
+            var list = new EntryList(entryType);
+
+            var type = entryType == 0 ? "Incomes" : "Expenses";
+
+            string query = "SELECT * FROM " + type + " WHERE UserId =" + userId;
+
+            DataSet MyDataSet = new DataSet();
+            try
+            {
+                System.Data.SqlClient.SqlDataAdapter MyDataAdapter = new System.Data.SqlClient.SqlDataAdapter(query, CnStr);
+
+                MyDataAdapter.Fill(MyDataSet);
+            }
+            catch (Exception ex)
+            {
+                return null;
+            }
```

Comment:
You don't have to catch exceptions here, we have exception handling everywhere in the website, so if anything goes wrong in this code, it will be caught and logged